kibana-reports/server/plugin.ts

```diff
@@ -19,7 +19,7 @@ import {
   CoreStart,
   Plugin,
   Logger,
-  IClusterClient,
+  ILegacyClusterClient,
```

Comment:
We revert back to legacy for 7.9?

Reply:
yes, the `IClusterClient` in kibana v7.9.1 is not fully supported for our usage.

Reply:
Got it